lib/browser/rpc-server.js

```diff
@@ -146,6 +146,30 @@ const throwRPCError = function (message) {
   throw error
 }
 
+const rendererMissingErrorMessage = function (meta, args, callIntoRenderer) {
+  let message = `Attempting to call a function in a renderer window that has been closed or released.` +
+    `\nFunction provided here: ${meta.location}`
+
+  if (!args || args.length === 0) return message
```

Comment:
`args` will never be passed as  `null` by the calling method so I think you can just use `arg.length === 0` here.